httpcore/_backends/asyncio.py

```diff
@@ -131,9 +131,20 @@ async def read(self, n: int, timeout: TimeoutDict) -> bytes:
         exc_map = {asyncio.TimeoutError: ReadTimeout, OSError: ReadError}
         async with self.read_lock:
             with map_exceptions(exc_map):
-                return await asyncio.wait_for(
-                    self.stream_reader.read(n), timeout.get("read")
-                )
+                try:
+                    return await asyncio.wait_for(
+                        self.stream_reader.read(n), timeout.get("read")
+                    )
+                except AttributeError:  # pragma: nocover
```

Comment:
Should we check that the attribute lookup that caused the error is indeed due to `resume_reading`?

```python
except AttributeError as exc:
    if "resume_reading" in str(exc):
        # Explanation...
        return b""
    raise
```